src/app/core/csp.service.ts

```diff
@@ -0,0 +1,27 @@
+import { DOCUMENT } from '@angular/common';
+import { Inject, Injectable } from '@angular/core';
+import { environment } from '../../environments/environment';
+
+
+@Injectable()
+export class CspService {
+  constructor(
+    @Inject(DOCUMENT) private doc: Document
+  ) { }
+
+  register() {
+    if (environment.name !== 'dev') {
+      const meta: HTMLMetaElement = this.doc.createElement('meta');
+      meta.setAttribute('http-equiv', 'Content-Security-Policy');
+      meta.setAttribute(
+        'content',
+        `default-src 'self';
+         child-src 'none';
+         font-src 'self' data:;
+         style-src 'unsafe-inline';
+         script-src 'self' 'unsafe-eval'`
+      );
```

Comment:
I'd also like to add connect-src with a whitelist of all domains we make XHRs to.

Reply:
* **content-src** - If I'm not mistaken, there are no any other domains that are allowed besides the localhost. If `connect-src` is missing, the value of `default-src` is used. So we're good.
* **form-action** - good call. Added.
* **plugin-types** - we have `object-src 'none';`
> To disallow all plugins, the object-src directive should be set to 'none' which will disallow plugins. The plugin-types directive is only used if you are allowing plugins with object-src at all.